doc/how-to-report-bugs.rst

```diff
@@ -0,0 +1,9 @@
+How to report Cabal bugs and deficiencies
+=========================================
+
+Please report any flaws or feature requests in the `bug
+tracker <https://github.com/haskell/cabal/issues>`__.
+
+For general discussion or queries email the libraries mailing list
+libraries@haskell.org. There is also a development mailing list
+cabal-devel@haskell.org.
```

Comment:
I understand this patch is not meant to modify content, but we need to integrate Matrix into the docs (and possibly axe `libraries`) sooner or later.

Reply:
I just saw that this information will as part of https://github.com/haskell/cabal/pull/9354/files